tasks/release.py

```diff
@@ -1101,3 +1101,62 @@ def create_github_release(_ctx, version, draft=True):
     )
 
     print(f"Link to the release note: {release.html_url}")
+
+
+from contextlib import contextmanager
+
+
+@contextmanager
+def agent_context(ctx, version: int):
+    REPO_AGENT6 = "../datadog-agent-6"
+
+    if version == 6:
+        # Need to go to the agent 6 repository
+
+        # Clone the repo if necessary
+        if not os.path.exists(REPO_AGENT6):
+            print('Cloning agent 6')
+            ctx.run(f"git clone git@github.com:DataDog/datadog-agent.git {REPO_AGENT6}")
```

Comment:
We could perhaps consider using [worktrees](https://git-scm.com/docs/git-worktree) for this as I expect them to be more lightweight. Doing a shallow clone might work too.